packages/block-library/src/rss/edit.js

```diff
@@ -119,6 +119,8 @@ class RSSEdit extends Component {
 							onChange={ ( value ) => setAttributes( { itemsToShow: value } ) }
 							min={ DEFAULT_MIN_ITEMS }
 							max={ DEFAULT_MAX_ITEMS }
+							allowReset={ true }
+							initialValue={ 5 }
```

Comment:
@aduth - this is similar to the heading level, and `enum` definition as this allows to define ranges. In this case, it would be great to access the default value which is defined in the PHP file. Once we migrate all blocks to use `block.json` definition, we could reference in theory the same value which is defined in `attributes`. However, given that hooks might modify this value, we should take that into account as well. It's interesting how complex it might get.

Reply:
okay. The attribute that holds the default changes each time that we move the slider. Since this gets called every time the slider changes, that value is irrelevant. So I decided to store the default value when we load the block in it's state.

Reply:
> @aduth - this is similar to #14689 and the heading level, and `enum` from definition as this allows to define ranges. In this case, it would be great to access the default value which is defined in the PHP file. Once we migrate all blocks to use `block.json` definition, we could reference in theory the same value which is defined in `attributes`. However, given that hooks might modify this value, we should take that into account as well. It's interesting how complex it might get.

We could also read directly from the active block definition, which would account for filtering:

```
wp.data.select( 'core/blocks' ).getBlockType( 'core/rss' ).attributes.itemsToShow.default
```